CMakeLists.txt

```diff
@@ -10,13 +10,8 @@ macro(build_mimick)
   set(cmake_configure_args -Wno-dev)
   list(APPEND cmake_configure_args -DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_install)
 
-  if(WIN32)
-    if(DEFINED CMAKE_GENERATOR)
-      list(APPEND cmake_configure_args -G ${CMAKE_GENERATOR})
-    endif()
-    if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "^(x86_|x86-|AMD|amd|x)64$")
-      list(APPEND cmake_configure_args -A x64)
-    endif()
```

Comment:
these 3 lines of code should be deleted, actually it **is** deleted in the PR.

I build ros2 from source on Windows by cmd like this:
```sh
colcon build --merge-install --cmake-args  -G "Visual Studio 17 2022" -A x64
```
Please note that I have already used `-A x64` in the cmake arguments, all projects inside ros2 will be built on platform of x64 and I don't need to specify it for each project individually. 

If in this repo's cmakelists.txt, `-A` is specified **again**, cmake will not work:
>CUSTOMBUILD : CMake error : Multiple -A options not allowed [C:\ros\rolling\build\mimick_vendor\mimick_vendor.vcxproj]

It's nice to see this PR actually removes the `-A`, please merge it.

Reply:
NICE

Reply:
Yep, looks good to me.  I will check on Windows early next week when I have it booted.

Reply:
Verified on Windows.